CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to Sage
+Thank you for considering contributing to Sage! We welcome all kinds of contributions. You don't need to be an AI expert to have meaningful impact. If you have any questions, reach out to us via [Discord](https://discord.gg/zbtZe7GcVU) or [email](mailto:founders@storia.ai).
```

Comment:
I wouldn't push for an email. Just have any questions posted as issues or we can just add them to the github discussions section.